lib/src/components/stack/Ancestor_Stack.res

```diff
@@ -1 +1,698 @@
+module Make = (Maker: Ancestor_StylesMaker.T) => {
+  module Base = Ancestor_Base.Make(Maker)
+  module Styles = Ancestor_Styles.Make(Maker)
 
+  %%private(
+    let defaultStyles = Maker.css(`
+      width: 100%;
+      flex-wrap: wrap;
+      display: flex;
+    `)
+
+    let createResponsiveStyles = (currentStyles, value) => {
+      let spacingInRem = value->Maker.unboxBreakpointValue->Styles.Spacing.make
+
+      Styles.mediaQuery(
+        currentStyles,
+        value,
+        `
+          &:before {
+            margin-top: -${spacingInRem};
+            content: "";
+          }
+
+          > * {
+            box-sizing: border-box;
+            margin-top: ${spacingInRem};
+          }
+          `,
+      )
+    }
+
+    let stack = (~spacing=?, ()) =>
+      spacing
+      ->Belt.Option.map(values =>
+        values
+        ->Js.Array2.sortInPlaceWith(Styles.sortBySize)
+        ->Belt.Array.reduce("", createResponsiveStyles)
+        ->Maker.css
+      )
+      ->Belt.Option.getWithDefault(defaultStyles)
```

Comment:
We don't need to pass `defaultStyles` as a default value here, this default value is just to make sure that we're passing an empty string when there is no value for the spacing prop.
```suggestion
->Belt.Option.getWithDefault("")
```